views/layouts/main.handlebars

```diff
@@ -1,78 +1,97 @@
+              <a
+                class='nav-link dropdown-toggle'
+                href='#'
+                role='button'
+                data-bs-toggle='dropdown'
+                aria-expanded='false'
+              >
+                Categories
+              </a>
+              <ul class='dropdown-menu'>
+                {{#each categories as |category|}}
+                  <a
+                    class='dropdown-item'
+                    href={{category.id}}
+                  >{{category.cat_name}}</a>
+                {{/each}}
+              </ul>
+            </li>
           </ul>
-        </li>
-        </ul>
-    </div>
-   <div class="mx-auto my-2 order-0 order-md-1 position-relative  "> 
-    <ul class="navbar-nav">
-    <li class="nav-item"><a href="#" class="nav-link">Profile</a></li>
-    <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
-    <li class="nav-item">{{#if logged_in}}
-        <a href="/profile" class="nav-link">Profile</a> |
-        <button class="no-button" id="logout">logout</button>
-        {{else}}
-        <a href="/login" class="nav-link">login</a>
-        {{/if}}</li>
-</ul>
-  
-         </div>
-   
-   
-</nav> 
-
-<div class="banner">
-  {{!-- <img src="pizza.jpg" alt="background" style="width: 100%; " /> --}}
-  <form class="form-inline my-2 my-lg-0">
-      <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
-      <button class="btn  my-2 my-sm-0 search-btn" type="submit"><i class="fa fa-search"></i></button>
-    </form>
-</div>
-</header>
-{{{ body }}}   
-  {{#if logged_in}}
-  <script src="/js/logout.js"></script>
-  {{/if}}    
-</main>
-  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-HwwvtgBNo3bZJJLYd8oVXjrBZt8cqVSpeBNS5n7C8IVInixGAoxmnlMuBnhbgrkm" crossorigin="anonymous"></script>
+        </div>
+        <div class='mx-auto my-2 order-0 order-md-1 position-relative'>
+          <ul class='navbar-nav'>
+            <li class='nav-item'><a href='#' class='nav-link'>Profile</a></li>
+            <li class='nav-item'><a href='#' class='nav-link'>Contact</a></li>
+            <li class='nav-item'>{{#if logged_in}}
+                <a href='/profile' class='nav-link'>Profile</a>
+                <a href="*" class='nav-link' id='logout'>logout</a>
+              {{else}}
+                <a href='/login' class='nav-link'>login</a>
+              {{/if}}</li>
+          </ul>
+        </div>
+      </nav>
+      <div class='banner'>
+        <img src="/banner.jpg" alt="yummy pizza">
+        <form class='form-inline my-2 my-lg-0'>
+          <input
+            class='form-control mr-sm-2'
+            type='search'
+            placeholder='Search'
+            aria-label='Search'
+          />
+          <button class='btn my-2 my-sm-0 search-btn' type='submit'><i
+              class='fa fa-search'
+            ></i></button>
+        </form>
+      </div>
+    </header>
```

Comment:
add div with class grid-container to apply on all pages

Reply:
applied in later push